core/backend/src/BriefcaseManager.ts

```diff
@@ -219,7 +219,14 @@ export class BriefcaseManager {
     } catch (error: unknown) {
       if (arg.accessToken && arg.briefcaseId === undefined)
         await this.releaseBriefcase(arg.accessToken, { briefcaseId, iModelId: arg.iModelId });
-
+      if (IModelJsFs.existsSync(fileName)) {
+        try {
+          IModelJsFs.unlinkSync(fileName);
+          Logger.logInfo(loggerCategory, `Deleted ${fileName}`);
```

Comment:
Consider amending your log messages to indicate why you tried to delete the briefcase.

Reply:
Resolved in [commit 3a4d3784b2d4b92344abb4026ed8d9cb08ad6db2](https://github.com/iTwin/itwinjs-core/pull/6805/commits/3a4d3784b2d4b92344abb4026ed8d9cb08ad6db2)